components/html/modules/content-patterns/_typography.html

```diff
@@ -0,0 +1,50 @@
+				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Debitis impedit suscipit cum ea cumque reiciendis laudantium sed, aut unde optio molestiae non aliquam officia aliquid est earum minus qui nulla. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Obcaecati aliquid, aspernatur exercitationem. Explicabo, impedit repudiandae maxime. Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime.</p>
+				<h2>H2 Archivo Narrow 32px #98700d</h2>
+				<p>Explicabo, impedit repudiandae maxime.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Debitis impedit suscipit cum ea cumque reiciendis laudantium sed, aut unde optio molestiae non aliquam officia aliquid est earum minus qui nulla. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Obcaecati aliquid, aspernatur exercitationem. Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime.</p>
+				<h3>H3 Archivo Narrow Bold 28px #707070</h3>
+				<p>Obcaecati aliquid, aspernatur exercitationem. Explicabo, impedit repudiandae maxime. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Debitis impedit suscipit cum ea cumque reiciendis laudantium sed, aut unde optio molestiae non aliquam officia aliquid est earum minus qui nulla. Lorem ipsum dolor sit amet, consectetur adipisicing elit.  Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime.</p>
+				<h4>H4 Archivo Bold 24px Caps #98700d</h4>
+				<p>Dolor sit amet, consectetur adipisicing elit. Obcaecati aliquid, aspernatur exercitationem. Explicabo, impedit repudiandae maxime. Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime. <a href="#">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</a> Debitis impedit suscipit cum ea cumque reiciendis laudantium sed, aut unde optio molestiae non aliquam officia aliquid est earum minus qui nulla.</p>
+				<aside class="highlight">
+					<h3 class="highlight__title">Archivo Bold 22px Caps #000000</h3>
+					<p class="highlight__content">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aspernatur ipsum perspiciatis veniam, rem totam optio distinctio. Porro, nisi in unde, nemo nulla totam corporis beatae natus, tempora hic aliquid. Voluptatum.</p>
+				</aside>
+				<h5>H5 Archivo Bold 22px Caps #000000</h5>
+				<p>Explicabo, impedit repudiandae maxime. Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime.</p>
+				<h6>H6 Archivo Bold 18px Caps #000000</h6>
+				<p>Aliquam officia aliquid est earum minus qui nulla. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Obcaecati aliquid, aspernatur exercitationem. Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime.</p>
+				<aside class="pullQuote">
```

Comment:
This would represent the default style for a pull quote - a gold bar on the left.